clients/fossid-webapp/src/main/kotlin/EntityResponseBody.kt

```diff
@@ -35,6 +35,10 @@ typealias MapResponseBody<T> = EntityResponseBody<Map<String, T>>
 
 typealias PolymorphicResponseBody<T> = EntityResponseBody<PolymorphicList<T>>
 
+typealias PolymorphicDataResponseBody<T> = EntityResponseBody<PolymorphicData<T>>
```

Comment:
Commit message nit: We usually say "Fixes" (like in the next commit) instead of "Fix".